session/multivac.py

```diff
@@ -28,16 +24,20 @@ class Multivac:
     """
     The Multivac class starts an environment and an agent in that environment.
     """
-    def __init__(self, environment_name, agent_name, num_steps, redis_port, image_height=DEFAULT_IMAGE_HEIGHT,
-                 image_width=DEFAULT_IMAGE_WIDTH, video_fps=1, display_video=False):
+
+    def __init__(self, environment_name, agent_name, num_steps, redis_port, video_fps=1, display_video=False):
         """
-        Initialize the Multivac.
+        Initialize the Multivac. This involves,
+          1. Open a redis client and setting up an action and observation buffer. This establishes an exchange
+             protocol between the Multivac and a ConnectionClient
+          2. Gather device metadata from the observation buffer
```

Comment:
Change this to `Gather an initial observation from the observation buffer to gather metadata information`